### PR TITLE
chore: enable CLI to work with Multi-Tenant Realtime

### PR DESCRIPTION
**Pull request by w3b6x9**
## What kind of change does this PR introduce?

chore

## What is the new behavior?

CLI works with Realtime v2, including Broadcast and Presence.

### CODE REVIEW

File: internal/start/templates/kong_config [hunk 1]
```diff
@@ -76,7 +76,7 @@ services:
               - "Content-Profile: graphql_public"
   - name: realtime-v1
     _comment: "Realtime: /realtime/v1/* -> ws://realtime:4000/socket/*"
-    url: http://supabase_realtime_{{ .ProjectId }}:4000/socket/
+    url: http://realtime.supabase_realtime_{{ .ProjectId }}:4000/socket
```
Comment (w3b6x9) [inline internal/start/templates/kong_config:79]: This is necessary b/c Realtime grabs the tenant id from the subdomain.

File: internal/utils/config.go [hunk 2]
```diff
@@ -165,7 +165,7 @@ func LoadConfigFS(fsys afero.Fs) error {
 			KongId = "supabase_kong_" + Config.ProjectId
 			GotrueId = "supabase_auth_" + Config.ProjectId
 			InbucketId = "supabase_inbucket_" + Config.ProjectId
-			RealtimeId = "supabase_realtime_" + Config.ProjectId
+			RealtimeId = "realtime.supabase_realtime_" + Config.ProjectId
```
Comment (w3b6x9) [inline internal/utils/config.go:168]: Support https://github.com/supabase/cli/pull/594/files#r1023417541.

File: internal/utils/misc.go [hunk 3]
```diff
@@ -65,8 +65,8 @@ SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE datname = '%[1]s';
 DO 'BEGIN WHILE (
 	SELECT COUNT(*) FROM pg_replication_slots WHERE database = ''%[1]s''
 ) > 0 LOOP END LOOP; END';`
-	AnonKey        = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24ifQ.625_WdcF3KHqz5amU0x2X5WWHP-OEs_4qj0ssLNHzTs"
-	ServiceRoleKey = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6InNlcnZpY2Vfcm9sZSJ9.vI9obAHOGyVVKa3pD--kJlyxp-Z2zV9UUMAhKpNLAcU"
+	AnonKey        = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24iLCJleHAiOjE5ODM4MTI5OTZ9.CRXP1A7WOeoJeXxjNni43kdQwgnWNReilDMblYTn_I0"
+	ServiceRoleKey = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImV4cCI6MTk4MzgxMjk5Nn0.EGIM96RAZx35lJzdJsyH-qQwv8Hdp7fsn3W0YpN81IU"
```
Comment (w3b6x9) [inline internal/utils/misc.go:69]: Realtime requires `exp` in the token.

File: internal/utils/misc.go [hunk 4]
```diff
@@ -103,6 +103,7 @@ var (
 		"pgbouncer",
 		"pgsodium",
 		"pgsodium_masks",
+		"_realtime",
```
Comment (w3b6x9) [inline internal/utils/misc.go:106]: This is a workaround for Realtime's DB that keeps track of all tenants and their database credentials.

**Review by sweatybridge — APPROVED** [review]
LGTM. Thanks for the helpful offline discussions. We can look into improving the hardcoded parts separately.

**Pull request merged**